modifiers.py

```diff
@@ -1,16 +1,15 @@
 class Modifier:
-    def __init__(self, caster, target, duration):
+    def __init__(self, caster, target, duration, damage=0):
```

Comment:
Nie każdy Modifier potrzebuje pola damage. 
Jeśli go nie potrzebuje to nie powinien go mieć, szkoda pamięci i bałaganu.
Pole damage niech ma dopiero obiekt tej klasy, która z tego skorzysta.

Reply:
poprawione